### ISSUE TITLE
listview automatically scrolls

### ISSUE THREAD

**Issue by renas** [RobotiumTech/robotium#269]
_From [koneru.s...@gmail.com](https://code.google.com/u/102882070009993887813/) on November 30, 2011 23:13:35_

What steps will reproduce the problem? 1. In my app, I was searching for ipod in search box(textview) and there are lots of items that show up in list view in SearchResultListActivity.
2. Noticed that the items keep scrolling automatically in the list view, even though I didn't do any search or requested for any action in that activity. What is the expected output? What do you see instead? The list shouldn't scroll rather should move on with next command. 
But the actual output is the list view keeps scrolling through all the items in the list and doesn't invoke next command that I quoted in testcase. What version of the product are you using? On what operating system? robotium - 3.0 Please provide any additional information below.

_Original issue: http://code.google.com/p/robotium/issues/detail?id=190_

**Comment by renas** [comment]
_From [koneru.s...@gmail.com](https://code.google.com/u/102882070009993887813/) on November 30, 2011 23:17:09_

one of the points that I missed mentioning is, I did set solo.waitForActivity() at different invocations in my test case. I am guessing this might be causing some issue. 
But as such I didn't invoke any solo.searchText("") to validate any text in that list view.

**Comment by renas** [comment]
_From [renasr...@gmail.com](https://code.google.com/u/108052454046732184536/) on November 30, 2011 23:23:52_

Can you please paste your test case? And explain were you are in the test case when the list appears.

**Comment by renas** [comment]
_From [koneru.s...@gmail.com](https://code.google.com/u/102882070009993887813/) on December 01, 2011 17:00:39_

My mistake.

I noticed, I have specified solo.searchText() in a activity to validate the content in the dialog box. But the dialog doesn't appear all the time. 
The next step is solo.goBack() which gets to previous activity that displays a list view, so the search continues in this page.

Sorry for the wrong question.
